<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AUDTopicsDlg</class>
 <widget class="QDialog" name="AUDTopicsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add|Update|Delete Topics</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>11</y>
     <width>671</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>Forum Topic Settings</string>
   </property>
   <widget class="QGroupBox" name="groupBox_Existing_Forum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>261</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Existing  Topics</string>
    </property>
    <widget class="QListWidget" name="listWidget_Existing_Topics">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="UNSelect_AUD_Topic">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>UnSelect All</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>99</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>         Topic Description</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Desc_AUD_Topic">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Admin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Need Administrator Approval</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Select_All_AUD_Topics">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_Email">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>491</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>         Existing Forum Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_Forum_Name_Existing">
       <property name="placeholderText">
        <string>Existing Forum Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>491</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Forum_Type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>        Topic Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Topic_name_Existing">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="AddNew_AUD_Topic">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>481</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel_AUD_Topic">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>480</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_AUD_Topic">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>481</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Delete">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
